perfmetrics/scripts/generate_files.py

```diff
@@ -7,29 +7,49 @@
 import argparse
 import configparser
 from datetime import datetime as dt
+import logging
 import os
 import subprocess
 from subprocess import Popen
 import sys
 
 OUTPUT_FILE = str(dt.now().isoformat()) + '.out'
-TEMPORARY_DIRECTORY = '/tmp/data_gen'
+TEMPORARY_DIRECTORY = './tmp/data_gen'
 BATCH_SIZE = 100
 
 
-def logmessage(message) -> None:
-  with open(OUTPUT_FILE, 'a') as out:
-    out.write(message)
-  print(message)
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s [%(levelname)s] %(message)s',
+    handlers=[logging.StreamHandler(sys.stdout)],
+)
+logger = logging.getLogger()
+
+
+def logmessage(message, output_format) -> None:
+  if output_format == 'output_file':
```

Comment:
the purpose of logging to file is since the creation of files takes longer time we wanted to log to file to avoid loosing messages on console.
In your case also the script execution will take time in case lot of files needs to created and we might execute the script in background in future. so its good to log to file in both the cases. 

So you need to take output_format as a param. just log to file and logger always. remove the print statement.

Reply:
resolved.